interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala

```diff
@@ -16,9 +16,11 @@ import java.util.{Currency, Locale, UUID}
   * is lack of conversions between String and java time API (ZoneId, ZoneOffset, etc.), it has only support for legacy
   * java Date/Calendar API
   */
-object NuConversionServiceFactory {
+object DefaultSpelConversionsProvider extends DefaultSpelConversionsProvider
 
-  def prepareConversionService: ConversionService = {
+class DefaultSpelConversionsProvider extends SpelConversionsProvider {
```

Comment:
Do we need both class and object? If they are not loaded via ServiceLoader?

Reply:
It is in case if someone would like to extends this class and add some additional covnersions